README.md

```diff
@@ -292,6 +292,68 @@ record allowance_accounts =
   , for_account: address }
 ```
 
+### allowances\(\)
+
+This function returns all of the allowances stored in `state.allowances` record.
+
+```sophia
+entrypoint allowances() : allowances
+```
+
+| return | type |
+| :--- | :--- |
+| allowances | map(allowance_accounts, int) |
+
+
+### allowance_for_caller\(\)
+
+This function will look up the allowances and return the allowed spendable amount from `from_account` for the transaction sender `Call.caller`. If there is no such allowance present result is `None`, otherwise `Some(int)` is returned with the allowance amount.
+
+```sophia
+entrypoint allowance_for_caller(from_account: address) : option(int)
+```
+
+| parameter | type |
+| :--- | :--- |
+| from_account| address |
+
+| return | type |
+| :--- | :--- |
+|  | option(int) |
+
+### change_allowance\(\)
+
+Allows to change the allowed spendable value for `for_account` with `value_change`
```

Comment:
I'd add more explanation around the change value, how it is applied, that it is added or substracted from the existing amount